src/ChaiExpectation.ts

```diff
@@ -12,32 +20,80 @@ type DeepMatchReturn = { to: {
 // type hack to get the ts to be happy
 type DeepMatch = (args: {}, errMsg?: string) => DeepMatchReturn;
 
-export class ChaiExpectation implements Expectation {
-  private chaiModule: typeof chai;
 
-  constructor(private subject: {}, private message?: string) {
-    this.chaiModule = require('chai');
```

Comment:
Changing this will `require` the chai dependency always. I know that for the moment the only supported expectation library is chai but just wanted to mention it.

Reply:
Good point. I think for now it should be fine as you mentioned. I figured that this should only be used in non-prod environments, so a little bit of bloat for clean code is fine. I'll look back at it and update later if need be.

Reply:
Cool, just wanted to make sure ;). Until other expectation libraries are added I don't think we should worry about this.